tests/Feature/HighlightTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Tests\Feature;
+
+use Aic\Hub\Foundation\Testing\FeatureTestCase as BaseTestCase;
+
+class HighlightTest extends BaseTestCase
+{
+    public function test_highlighted_selections()
+    {
+        $this->markTestIncomplete("Verify highlighted selections are displayed.");
```

Comment:
Some of the language that we use to identify models and attributes is confusing here. "Highlight" and "Selection" both refer to the same type of content. When the site was first created we used to call [Highlights](https://www.artic.edu/highlights) "Selections," but later renamed them to Highlights. I think the code here represents a moment in time where part of the codebase called them "Highlights" and other parts of the codebases still called them "Selections," e.g., the table was still called "selections" while the column name "highlight_type" refers to the new nomenclature. See WEB-581 and WEB-1075 for more context.

I think what you can test here is that there are no records with a `null` highlight_type column.

Reply:
Yeah, after looking at the corresponding seeder to this, I believe I was mistaken in removing it. And this test should probably be something like:
- navigate to url that displays these highlights
- iterate over highlights to make sure they are all displayed

Reply:
To summarize our discussion yesterday: Highlights are content that are entered and managed by the CMS, and the _content_ doesn't need to be test in our code. We can remove this seeder and test and think about adding it back based around features we may introduce in the future.